front_end/src/app/admin/events/family-finder/family-finder.component.ts

```diff
@@ -7,6 +8,9 @@ import { Contact } from '../../../shared/models';
 import { AdminService } from '../../admin.service';
 import { HeaderService } from '../../header/header.service';
 import { ApiService } from '../../../shared/services';
+const phoneRegex = /^\d{3}-\d{3}-\d{4}$/;
+const nameRegex = /^[a-zA-Z]*$/;
+const emailRegex = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
It makes sure there isn't a number in it and at least two character long, but if we nail it down then we'll have to change this regex as new TLD's are released... theres hundreds of valid ones currently - http://data.iana.org/TLD/tlds-alpha-by-domain.txt